app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WelcomeController < ApplicationController
+  def index
+    @movies = Movie.order(rank: :desc).limit(10)
+    @books = Book.order(rank: :desc).limit(10)
+    @albums = Album.order(rank: :desc).limit(10)
```

Comment:
Have you discovered/learned about scopes yet? This would be good opportunity to check them out.